.github/workflows/build-world.yaml

```diff
@@ -0,0 +1,96 @@
+name: Build the world
+
+on:
+  workflow_dispatch:
+    
+jobs:
+  build:
+    name: Build OS packages
+    runs-on:
+      - self-hosted
+      - ${{ matrix.arch }}
```

Comment:
Are we sure this works?

Reply:
I stole this from: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-using-a-multi-dimension-matrix